README.md

```diff
@@ -19,6 +19,12 @@ Follow the instructions on [get started with GitHub Pages](//pages.github.com/).
 
 Fork [reveal-jekyll](//github.com/tasmo/reveal-jekyll) and name it with your user or organisation name like `<yourname>.github.io`.
 
+You can also host as a [project page](https://help.github.com/articles/user-organization-and-project-pages/#project-pages):
+
+- Clone this repo with a name like `mytalk`
+- Delete the `gh-pages` branch, create a new `gh-pages` branch off of `master`, and set `gh-pages` as the default branch. You can do all this through the Github web UI (TODO: details)
```

Comment:
Well done, just some suggestions:

+ Delete the cloned `gh-pages` branch, create a new `gh-pages` branch based on `master` and set it as your new default branch.
~~~shell
git checkout master
git branch -d gh-pages
git push origin :gh-pages
git branch gh-pages
git checkout gh-pages
git remote set-head origin gh-pages
~~~